app/modules/assets/models.py

```diff
@@ -278,7 +288,7 @@ def set_derived_meta(self):
         meta = self.meta if self.meta else {}
         meta['derived'] = dmeta
         self.meta = meta
-        log.debug(f'setting meta.derived to {dmeta}')
+        # log.debug(f'setting meta.derived to {dmeta}')
```

Comment:
delete rather than comment out unused trace. I agree that this is not needed.

Reply:
I have reverted this change.  The debug logging is going to stay.